### ISSUE TITLE
update heroku deployment to reflect dependency changes

### ISSUE THREAD

**Issue by bengolder** [codeforamerica/pdfhook#74]
Now that this no longer depends on pdftk, the heroku and govcloud deployment configuration should be updated. If someone tried to deploy this in its current state, it might break.

**Comment by brennv** [comment]
Does the govcloud config refer to cloudfoundry?

Also can we add this service? https://requires.io/

**Comment by bengolder** [comment]
Yes, I thing govcloud is a specific running version of cloudfoundry.

**Comment by bengolder** [comment]
And requires.io looks great! Yes!
